src/fciqmc.jl

```diff
@@ -465,8 +465,10 @@ function fciqmc_step!(Ĥ, dv, shift, dτ, pnorm, w, m = 1.0;
     empty!(w) # clear working memory
     # call fciqmc_col!() on every entry of `v` and add the stats returned by
     # this function:
-    stats = sum(p-> SVector(fciqmc_col!(w, Ĥ, p.first, p.second, shift, dτ)),
-      pairs(v))
+    stats = SVector(0, 0, 0, 0, 0)
+    for (k, v) in pairs(v)
+        stats += SVector(fciqmc_col!(w, Ĥ, k, v, shift, dτ))
+    end
```

Comment:
`stats` may be complex, so the initialisation should be more general. See how it is done for the threaded version:

```
function fciqmc_step!(Ĥ, dv, shift, dτ, pnorm, ws::NTuple{NT,W}, f::Float64;
      m_strat::MemoryStrategy = NoMemory()
    ) where {NT,W}
    # multithreaded version; should also work with MPI
    @assert NT == nthreads() "`nthreads()` not matching dimension of `ws`"
    v = localpart(dv)
    statss = allocate_statss(v, NT)

```